libraries/AdaptiveExpressions/FunctionUtils.cs

```diff
@@ -1413,25 +1413,17 @@ internal static EvaluateExpressionDelegate SortBy(bool isDescending)
                return (result, error);
            };
 
-        private static (object, string) ParseISOTimestamp(string timeStamp, Func<DateTime, (object, string)> transform = null)
+        private static (object, string) ParseTimestamp(string timeStamp, Func<DateTime, (object, string)> transform = null)
```

Comment:
I will still call this ParseISOTimeStamp, because it is. The previous checking is a subset of ISO format.